zetaclient/zetabridge/zetacore_bridge.go

```diff
@@ -276,3 +274,8 @@ func (b *ZetaCoreBridge) Pause() {
 func (b *ZetaCoreBridge) Unpause() {
 	b.pause <- struct{}{}
 }
+
+func (b *ZetaCoreBridge) EnableMockSDKClient(client rpcclient.Client) {
+	b.mockSDKClient = client
+	b.cfg.UseMockSDKClient = true
```

Comment:
Would it be better to bring the test flag `UseMockSDKClient` from config file up to `ZetaCoreBridge` struct, same level as the `mockSDKClient`. Then we determine if mock client needs to be used in the `GetContext` func according to the flag?